[2025-05-11T03:52:23.102+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-11T03:52:23.157+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [queued]>
[2025-05-11T03:52:23.169+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [queued]>
[2025-05-11T03:52:23.170+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-05-11T03:52:23.190+0700] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): run_lstm_dbt> on 2025-05-08 11:30:00+00:00
[2025-05-11T03:52:23.203+0700] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bbca_lstm_prediction', 'run_lstm_dbt', 'scheduled__2025-05-08T11:30:00+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/stock_predictions_bbca.py', '--cfg-path', '/tmp/tmpw192aocc']
[2025-05-11T03:52:23.206+0700] {standard_task_runner.py:91} INFO - Job 104: Subtask run_lstm_dbt
[2025-05-11T03:52:23.218+0700] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1445) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-11T03:52:23.220+0700] {standard_task_runner.py:63} INFO - Started process 1477 to run task
[2025-05-11T03:52:23.323+0700] {task_command.py:426} INFO - Running <TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [running]> on host a61c3a2f78e1
[2025-05-11T03:52:23.505+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bbca_lstm_prediction' AIRFLOW_CTX_TASK_ID='run_lstm_dbt' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T11:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-08T11:30:00+00:00'
[2025-05-11T03:52:23.507+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-11T03:52:23.530+0700] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-11T03:52:23.531+0700] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt && dbt run --profiles-dir /opt/***/dbt --select stg_stock_predictions fct_stock_predictions lstm_performance_metrics stock_prediction_dashboard']
[2025-05-11T03:52:23.554+0700] {subprocess.py:86} INFO - Output:
[2025-05-11T03:52:27.250+0700] {subprocess.py:93} INFO - [0m20:52:27  Running with dbt=1.9.4
[2025-05-11T03:52:27.731+0700] {subprocess.py:93} INFO - [0m20:52:27  Registered adapter: postgres=1.9.0
[2025-05-11T03:52:28.995+0700] {subprocess.py:93} INFO - [0m20:52:28  Found 14 models, 3 data tests, 6 sources, 434 macros
[2025-05-11T03:52:29.003+0700] {subprocess.py:93} INFO - [0m20:52:29
[2025-05-11T03:52:29.005+0700] {subprocess.py:93} INFO - [0m20:52:29  Concurrency: 4 threads (target='dev')
[2025-05-11T03:52:29.007+0700] {subprocess.py:93} INFO - [0m20:52:29
[2025-05-11T03:52:29.318+0700] {subprocess.py:93} INFO - [0m20:52:29  1 of 4 START sql view model public_staging.stg_stock_predictions ............... [RUN]
[2025-05-11T03:52:29.559+0700] {subprocess.py:93} INFO - [0m20:52:29  1 of 4 OK created sql view model public_staging.stg_stock_predictions .......... [[32mCREATE VIEW[0m in 0.23s]
[2025-05-11T03:52:29.567+0700] {subprocess.py:93} INFO - [0m20:52:29  2 of 4 START sql table model public_core.fct_stock_predictions ................. [RUN]
[2025-05-11T03:52:29.700+0700] {subprocess.py:93} INFO - [0m20:52:29  2 of 4 ERROR creating sql table model public_core.fct_stock_predictions ........ [[31mERROR[0m in 0.13s]
[2025-05-11T03:52:29.712+0700] {subprocess.py:93} INFO - [0m20:52:29  3 of 4 SKIP relation public_analytics.lstm_performance_metrics ................. [[33mSKIP[0m]
[2025-05-11T03:52:29.714+0700] {subprocess.py:93} INFO - [0m20:52:29  4 of 4 SKIP relation public_analytics.stock_prediction_dashboard ............... [[33mSKIP[0m]
[2025-05-11T03:52:29.757+0700] {subprocess.py:93} INFO - [0m20:52:29
[2025-05-11T03:52:29.758+0700] {subprocess.py:93} INFO - [0m20:52:29  Finished running 3 table models, 1 view model in 0 hours 0 minutes and 0.75 seconds (0.75s).
[2025-05-11T03:52:29.875+0700] {subprocess.py:93} INFO - [0m20:52:29
[2025-05-11T03:52:29.877+0700] {subprocess.py:93} INFO - [0m20:52:29  [31mCompleted with 1 error, 0 partial successes, and 0 warnings:[0m
[2025-05-11T03:52:29.879+0700] {subprocess.py:93} INFO - [0m20:52:29
[2025-05-11T03:52:29.881+0700] {subprocess.py:93} INFO - [0m20:52:29    Database Error in model fct_stock_predictions (models/marts/core/fct_stock_predictions.sql)
[2025-05-11T03:52:29.882+0700] {subprocess.py:93} INFO -   subquery in FROM must have an alias
[2025-05-11T03:52:29.882+0700] {subprocess.py:93} INFO -   LINE 29:     FROM (
[2025-05-11T03:52:29.883+0700] {subprocess.py:93} INFO -                     ^
[2025-05-11T03:52:29.884+0700] {subprocess.py:93} INFO -   HINT:  For example, FROM (SELECT ...) [AS] foo.
[2025-05-11T03:52:29.885+0700] {subprocess.py:93} INFO -   compiled code at target/run/idx_stock/models/marts/core/fct_stock_predictions.sql
[2025-05-11T03:52:29.885+0700] {subprocess.py:93} INFO - [0m20:52:29
[2025-05-11T03:52:29.886+0700] {subprocess.py:93} INFO - [0m20:52:29  Done. PASS=1 WARN=0 ERROR=1 SKIP=2 TOTAL=4
[2025-05-11T03:52:31.678+0700] {subprocess.py:97} INFO - Command exited with return code 1
[2025-05-11T03:52:31.679+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-11T03:52:31.701+0700] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-05-11T03:52:31.707+0700] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=bbca_lstm_prediction, task_id=run_lstm_dbt, run_id=scheduled__2025-05-08T11:30:00+00:00, execution_date=20250508T113000, start_date=20250510T205223, end_date=20250510T205231
[2025-05-11T03:52:31.725+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 104 for task run_lstm_dbt (Bash command failed. The command returned a non-zero exit code 1.; 1477)
[2025-05-11T03:52:31.740+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-05-11T03:52:31.764+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-11T03:52:31.770+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-05-11T16:57:53.781+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-11T16:57:53.825+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [queued]>
[2025-05-11T16:57:53.837+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [queued]>
[2025-05-11T16:57:53.838+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-05-11T16:57:53.858+0700] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): run_lstm_dbt> on 2025-05-08 11:30:00+00:00
[2025-05-11T16:57:53.873+0700] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3956) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-11T16:57:53.873+0700] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bbca_lstm_prediction', 'run_lstm_dbt', 'scheduled__2025-05-08T11:30:00+00:00', '--job-id', '161', '--raw', '--subdir', 'DAGS_FOLDER/stock_predictions_bbca.py', '--cfg-path', '/tmp/tmpgimbt4nj']
[2025-05-11T16:57:53.875+0700] {standard_task_runner.py:63} INFO - Started process 3965 to run task
[2025-05-11T16:57:53.876+0700] {standard_task_runner.py:91} INFO - Job 161: Subtask run_lstm_dbt
[2025-05-11T16:57:53.956+0700] {task_command.py:426} INFO - Running <TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [running]> on host 615e6c656dab
[2025-05-11T16:57:54.093+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bbca_lstm_prediction' AIRFLOW_CTX_TASK_ID='run_lstm_dbt' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T11:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-08T11:30:00+00:00'
[2025-05-11T16:57:54.095+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-11T16:57:54.109+0700] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-11T16:57:54.110+0700] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt && dbt run --profiles-dir /opt/***/dbt --select stg_stock_predictions fct_stock_predictions lstm_performance_metrics stock_prediction_dashboard']
[2025-05-11T16:57:54.130+0700] {subprocess.py:86} INFO - Output:
[2025-05-11T16:57:57.203+0700] {subprocess.py:93} INFO - [0m09:57:57  Running with dbt=1.9.4
[2025-05-11T16:57:57.650+0700] {subprocess.py:93} INFO - [0m09:57:57  Registered adapter: postgres=1.9.0
[2025-05-11T16:57:58.783+0700] {subprocess.py:93} INFO - [0m09:57:58  Found 14 models, 3 data tests, 6 sources, 434 macros
[2025-05-11T16:57:58.799+0700] {subprocess.py:93} INFO - [0m09:57:58
[2025-05-11T16:57:58.801+0700] {subprocess.py:93} INFO - [0m09:57:58  Concurrency: 4 threads (target='dev')
[2025-05-11T16:57:58.803+0700] {subprocess.py:93} INFO - [0m09:57:58
[2025-05-11T16:57:59.141+0700] {subprocess.py:93} INFO - [0m09:57:59  1 of 4 START sql view model public_staging.stg_stock_predictions ............... [RUN]
[2025-05-11T16:57:59.398+0700] {subprocess.py:93} INFO - [0m09:57:59  1 of 4 OK created sql view model public_staging.stg_stock_predictions .......... [[32mCREATE VIEW[0m in 0.25s]
[2025-05-11T16:57:59.405+0700] {subprocess.py:93} INFO - [0m09:57:59  2 of 4 START sql table model public_core.fct_stock_predictions ................. [RUN]
[2025-05-11T16:58:00.115+0700] {subprocess.py:93} INFO - [0m09:58:00  2 of 4 OK created sql table model public_core.fct_stock_predictions ............ [[32mSELECT 1[0m in 0.71s]
[2025-05-11T16:58:00.124+0700] {subprocess.py:93} INFO - [0m09:58:00  3 of 4 START sql table model public_analytics.lstm_performance_metrics ......... [RUN]
[2025-05-11T16:58:00.126+0700] {subprocess.py:93} INFO - [0m09:58:00  4 of 4 START sql table model public_analytics.stock_prediction_dashboard ....... [RUN]
[2025-05-11T16:58:00.317+0700] {subprocess.py:93} INFO - [0m09:58:00  3 of 4 OK created sql table model public_analytics.lstm_performance_metrics .... [[32mSELECT 1[0m in 0.19s]
[2025-05-11T16:58:04.263+0700] {subprocess.py:93} INFO - [0m09:58:04  4 of 4 OK created sql table model public_analytics.stock_prediction_dashboard .. [[32mSELECT 53599[0m in 4.13s]
[2025-05-11T16:58:04.308+0700] {subprocess.py:93} INFO - [0m09:58:04
[2025-05-11T16:58:04.310+0700] {subprocess.py:93} INFO - [0m09:58:04  Finished running 3 table models, 1 view model in 0 hours 0 minutes and 5.50 seconds (5.50s).
[2025-05-11T16:58:04.427+0700] {subprocess.py:93} INFO - [0m09:58:04
[2025-05-11T16:58:04.429+0700] {subprocess.py:93} INFO - [0m09:58:04  [32mCompleted successfully[0m
[2025-05-11T16:58:04.431+0700] {subprocess.py:93} INFO - [0m09:58:04
[2025-05-11T16:58:04.433+0700] {subprocess.py:93} INFO - [0m09:58:04  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2025-05-11T16:58:06.167+0700] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-11T16:58:06.168+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-11T16:58:06.214+0700] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bbca_lstm_prediction, task_id=run_lstm_dbt, run_id=scheduled__2025-05-08T11:30:00+00:00, execution_date=20250508T113000, start_date=20250511T095753, end_date=20250511T095806
[2025-05-11T16:58:06.266+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-11T16:58:06.287+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-11T16:58:06.294+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-05-12T03:57:24.369+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-12T03:57:24.456+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [queued]>
[2025-05-12T03:57:24.481+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [queued]>
[2025-05-12T03:57:24.482+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-05-12T03:57:24.518+0700] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): run_lstm_dbt> on 2025-05-08 11:30:00+00:00
[2025-05-12T03:57:24.545+0700] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bbca_lstm_prediction', 'run_lstm_dbt', 'scheduled__2025-05-08T11:30:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/stock_predictions_bbca.py', '--cfg-path', '/tmp/tmpvn92bkoi']
[2025-05-12T03:57:24.552+0700] {standard_task_runner.py:91} INFO - Job 48: Subtask run_lstm_dbt
[2025-05-12T03:57:24.565+0700] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4194) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-12T03:57:24.567+0700] {standard_task_runner.py:63} INFO - Started process 4216 to run task
[2025-05-12T03:57:24.708+0700] {task_command.py:426} INFO - Running <TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [running]> on host b08a3a418dc4
[2025-05-12T03:57:24.959+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bbca_lstm_prediction' AIRFLOW_CTX_TASK_ID='run_lstm_dbt' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T11:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-08T11:30:00+00:00'
[2025-05-12T03:57:24.962+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-12T03:57:24.992+0700] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-12T03:57:24.994+0700] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt && dbt run --profiles-dir /opt/***/dbt --select stg_stock_predictions fct_stock_predictions lstm_performance_metrics stock_prediction_dashboard']
[2025-05-12T03:57:25.029+0700] {subprocess.py:86} INFO - Output:
[2025-05-12T03:57:31.443+0700] {subprocess.py:93} INFO - [0m20:57:31  Running with dbt=1.9.4
[2025-05-12T03:57:32.997+0700] {subprocess.py:93} INFO - [0m20:57:32  Registered adapter: postgres=1.9.0
[2025-05-12T03:57:36.077+0700] {subprocess.py:93} INFO - [0m20:57:36  Found 14 models, 3 data tests, 6 sources, 434 macros
[2025-05-12T03:57:36.090+0700] {subprocess.py:93} INFO - [0m20:57:36
[2025-05-12T03:57:36.094+0700] {subprocess.py:93} INFO - [0m20:57:36  Concurrency: 4 threads (target='dev')
[2025-05-12T03:57:36.098+0700] {subprocess.py:93} INFO - [0m20:57:36
[2025-05-12T03:57:36.769+0700] {subprocess.py:93} INFO - [0m20:57:36  1 of 4 START sql view model public_staging.stg_stock_predictions ............... [RUN]
[2025-05-12T03:57:37.350+0700] {subprocess.py:93} INFO - [0m20:57:37  1 of 4 OK created sql view model public_staging.stg_stock_predictions .......... [[32mCREATE VIEW[0m in 0.56s]
[2025-05-12T03:57:37.374+0700] {subprocess.py:93} INFO - [0m20:57:37  2 of 4 START sql table model public_core.fct_stock_predictions ................. [RUN]
[2025-05-12T03:57:39.046+0700] {subprocess.py:93} INFO - [0m20:57:39  2 of 4 OK created sql table model public_core.fct_stock_predictions ............ [[32mSELECT 1[0m in 1.66s]
[2025-05-12T03:57:39.063+0700] {subprocess.py:93} INFO - [0m20:57:39  3 of 4 START sql table model public_analytics.lstm_performance_metrics ......... [RUN]
[2025-05-12T03:57:39.069+0700] {subprocess.py:93} INFO - [0m20:57:39  4 of 4 START sql table model public_analytics.stock_prediction_dashboard ....... [RUN]
[2025-05-12T03:57:39.381+0700] {subprocess.py:93} INFO - [0m20:57:39  3 of 4 OK created sql table model public_analytics.lstm_performance_metrics .... [[32mSELECT 0[0m in 0.30s]
[2025-05-12T03:57:46.393+0700] {subprocess.py:93} INFO - [0m20:57:46  4 of 4 OK created sql table model public_analytics.stock_prediction_dashboard .. [[32mSELECT 53599[0m in 7.31s]
[2025-05-12T03:57:46.507+0700] {subprocess.py:93} INFO - [0m20:57:46
[2025-05-12T03:57:46.510+0700] {subprocess.py:93} INFO - [0m20:57:46  Finished running 3 table models, 1 view model in 0 hours 0 minutes and 10.41 seconds (10.41s).
[2025-05-12T03:57:46.678+0700] {subprocess.py:93} INFO - [0m20:57:46
[2025-05-12T03:57:46.681+0700] {subprocess.py:93} INFO - [0m20:57:46  [32mCompleted successfully[0m
[2025-05-12T03:57:46.684+0700] {subprocess.py:93} INFO - [0m20:57:46
[2025-05-12T03:57:46.687+0700] {subprocess.py:93} INFO - [0m20:57:46  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2025-05-12T03:57:49.199+0700] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-12T03:57:49.205+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-12T03:57:49.325+0700] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bbca_lstm_prediction, task_id=run_lstm_dbt, run_id=scheduled__2025-05-08T11:30:00+00:00, execution_date=20250508T113000, start_date=20250511T205724, end_date=20250511T205749
[2025-05-12T03:57:49.429+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-12T03:57:49.494+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-12T03:57:49.502+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-05-12T06:21:27.007+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-12T06:21:27.046+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [queued]>
[2025-05-12T06:21:27.056+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [queued]>
[2025-05-12T06:21:27.057+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-05-12T06:21:27.076+0700] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): run_lstm_dbt> on 2025-05-08 11:30:00+00:00
[2025-05-12T06:21:27.088+0700] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4060) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-12T06:21:27.088+0700] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bbca_lstm_prediction', 'run_lstm_dbt', 'scheduled__2025-05-08T11:30:00+00:00', '--job-id', '152', '--raw', '--subdir', 'DAGS_FOLDER/stock_predictions_bbca.py', '--cfg-path', '/tmp/tmp5l45yn60']
[2025-05-12T06:21:27.090+0700] {standard_task_runner.py:63} INFO - Started process 4069 to run task
[2025-05-12T06:21:27.091+0700] {standard_task_runner.py:91} INFO - Job 152: Subtask run_lstm_dbt
[2025-05-12T06:21:27.166+0700] {task_command.py:426} INFO - Running <TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [running]> on host 84b3813fe61b
[2025-05-12T06:21:27.282+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bbca_lstm_prediction' AIRFLOW_CTX_TASK_ID='run_lstm_dbt' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T11:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-08T11:30:00+00:00'
[2025-05-12T06:21:27.284+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-12T06:21:27.296+0700] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-12T06:21:27.297+0700] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt && dbt run --profiles-dir /opt/***/dbt --select stg_stock_predictions fct_stock_predictions lstm_performance_metrics stock_prediction_dashboard']
[2025-05-12T06:21:27.313+0700] {subprocess.py:86} INFO - Output:
[2025-05-12T06:21:30.043+0700] {subprocess.py:93} INFO - [0m23:21:30  Running with dbt=1.9.4
[2025-05-12T06:21:30.464+0700] {subprocess.py:93} INFO - [0m23:21:30  Registered adapter: postgres=1.9.0
[2025-05-12T06:21:31.464+0700] {subprocess.py:93} INFO - [0m23:21:31  Found 14 models, 3 data tests, 6 sources, 434 macros
[2025-05-12T06:21:31.471+0700] {subprocess.py:93} INFO - [0m23:21:31
[2025-05-12T06:21:31.473+0700] {subprocess.py:93} INFO - [0m23:21:31  Concurrency: 4 threads (target='dev')
[2025-05-12T06:21:31.475+0700] {subprocess.py:93} INFO - [0m23:21:31
[2025-05-12T06:21:31.751+0700] {subprocess.py:93} INFO - [0m23:21:31  1 of 4 START sql view model public_staging.stg_stock_predictions ............... [RUN]
[2025-05-12T06:21:31.961+0700] {subprocess.py:93} INFO - [0m23:21:31  1 of 4 OK created sql view model public_staging.stg_stock_predictions .......... [[32mCREATE VIEW[0m in 0.20s]
[2025-05-12T06:21:31.968+0700] {subprocess.py:93} INFO - [0m23:21:31  2 of 4 START sql table model public_core.fct_stock_predictions ................. [RUN]
[2025-05-12T06:21:32.688+0700] {subprocess.py:93} INFO - [0m23:21:32  2 of 4 OK created sql table model public_core.fct_stock_predictions ............ [[32mSELECT 1[0m in 0.72s]
[2025-05-12T06:21:32.696+0700] {subprocess.py:93} INFO - [0m23:21:32  3 of 4 START sql table model public_analytics.lstm_performance_metrics ......... [RUN]
[2025-05-12T06:21:32.699+0700] {subprocess.py:93} INFO - [0m23:21:32  4 of 4 START sql table model public_analytics.stock_prediction_dashboard ....... [RUN]
[2025-05-12T06:21:32.838+0700] {subprocess.py:93} INFO - [0m23:21:32  4 of 4 ERROR creating sql table model public_analytics.stock_prediction_dashboard  [[31mERROR[0m in 0.13s]
[2025-05-12T06:21:32.866+0700] {subprocess.py:93} INFO - [0m23:21:32  3 of 4 OK created sql table model public_analytics.lstm_performance_metrics .... [[32mSELECT 0[0m in 0.16s]
[2025-05-12T06:21:32.905+0700] {subprocess.py:93} INFO - [0m23:21:32
[2025-05-12T06:21:32.907+0700] {subprocess.py:93} INFO - [0m23:21:32  Finished running 3 table models, 1 view model in 0 hours 0 minutes and 1.43 seconds (1.43s).
[2025-05-12T06:21:33.012+0700] {subprocess.py:93} INFO - [0m23:21:33
[2025-05-12T06:21:33.014+0700] {subprocess.py:93} INFO - [0m23:21:33  [31mCompleted with 1 error, 0 partial successes, and 0 warnings:[0m
[2025-05-12T06:21:33.016+0700] {subprocess.py:93} INFO - [0m23:21:33
[2025-05-12T06:21:33.017+0700] {subprocess.py:93} INFO - [0m23:21:33    Database Error in model stock_prediction_dashboard (models/marts/analytics/stock_prediction_dashboard.sql)
[2025-05-12T06:21:33.018+0700] {subprocess.py:93} INFO -   column m.macd_histogram does not exist
[2025-05-12T06:21:33.019+0700] {subprocess.py:93} INFO -   LINE 28:         m.macd_histogram
[2025-05-12T06:21:33.019+0700] {subprocess.py:93} INFO -                    ^
[2025-05-12T06:21:33.020+0700] {subprocess.py:93} INFO -   compiled code at target/run/idx_stock/models/marts/analytics/stock_prediction_dashboard.sql
[2025-05-12T06:21:33.020+0700] {subprocess.py:93} INFO - [0m23:21:33
[2025-05-12T06:21:33.021+0700] {subprocess.py:93} INFO - [0m23:21:33  Done. PASS=3 WARN=0 ERROR=1 SKIP=0 TOTAL=4
[2025-05-12T06:21:34.627+0700] {subprocess.py:97} INFO - Command exited with return code 1
[2025-05-12T06:21:34.629+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-12T06:21:34.644+0700] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-05-12T06:21:34.650+0700] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=bbca_lstm_prediction, task_id=run_lstm_dbt, run_id=scheduled__2025-05-08T11:30:00+00:00, execution_date=20250508T113000, start_date=20250511T232127, end_date=20250511T232134
[2025-05-12T06:21:34.671+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 152 for task run_lstm_dbt (Bash command failed. The command returned a non-zero exit code 1.; 4069)
[2025-05-12T06:21:34.720+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-05-12T06:21:34.740+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-12T06:21:34.746+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-05-12T16:22:28.077+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-12T16:22:28.161+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [queued]>
[2025-05-12T16:22:28.178+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [queued]>
[2025-05-12T16:22:28.180+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-05-12T16:22:28.210+0700] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): run_lstm_dbt> on 2025-05-08 11:30:00+00:00
[2025-05-12T16:22:28.234+0700] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=5057) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-12T16:22:28.233+0700] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bbca_lstm_prediction', 'run_lstm_dbt', 'scheduled__2025-05-08T11:30:00+00:00', '--job-id', '131', '--raw', '--subdir', 'DAGS_FOLDER/stock_predictions_bbca.py', '--cfg-path', '/tmp/tmpfn3r9t1r']
[2025-05-12T16:22:28.238+0700] {standard_task_runner.py:63} INFO - Started process 5072 to run task
[2025-05-12T16:22:28.238+0700] {standard_task_runner.py:91} INFO - Job 131: Subtask run_lstm_dbt
[2025-05-12T16:22:28.378+0700] {task_command.py:426} INFO - Running <TaskInstance: bbca_lstm_prediction.run_lstm_dbt scheduled__2025-05-08T11:30:00+00:00 [running]> on host 8d034d8013d1
[2025-05-12T16:22:28.673+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bbca_lstm_prediction' AIRFLOW_CTX_TASK_ID='run_lstm_dbt' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T11:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-08T11:30:00+00:00'
[2025-05-12T16:22:28.676+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-12T16:22:28.702+0700] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-12T16:22:28.704+0700] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt && dbt run --profiles-dir /opt/***/dbt --select stg_stock_predictions fct_stock_predictions lstm_performance_metrics stock_prediction_dashboard']
[2025-05-12T16:22:28.739+0700] {subprocess.py:86} INFO - Output:
[2025-05-12T16:22:35.837+0700] {subprocess.py:93} INFO - [0m09:22:35  Running with dbt=1.9.4
[2025-05-12T16:22:37.058+0700] {subprocess.py:93} INFO - [0m09:22:37  Registered adapter: postgres=1.9.0
[2025-05-12T16:22:40.087+0700] {subprocess.py:93} INFO - [0m09:22:40  Found 14 models, 3 data tests, 6 sources, 434 macros
[2025-05-12T16:22:40.106+0700] {subprocess.py:93} INFO - [0m09:22:40
[2025-05-12T16:22:40.109+0700] {subprocess.py:93} INFO - [0m09:22:40  Concurrency: 4 threads (target='dev')
[2025-05-12T16:22:40.113+0700] {subprocess.py:93} INFO - [0m09:22:40
[2025-05-12T16:22:41.042+0700] {subprocess.py:93} INFO - [0m09:22:41  1 of 4 START sql view model public_staging.stg_stock_predictions ............... [RUN]
[2025-05-12T16:22:41.706+0700] {subprocess.py:93} INFO - [0m09:22:41  1 of 4 OK created sql view model public_staging.stg_stock_predictions .......... [[32mCREATE VIEW[0m in 0.65s]
[2025-05-12T16:22:41.720+0700] {subprocess.py:93} INFO - [0m09:22:41  2 of 4 START sql table model public_core.fct_stock_predictions ................. [RUN]
[2025-05-12T16:22:42.694+0700] {subprocess.py:93} INFO - [0m09:22:42  2 of 4 OK created sql table model public_core.fct_stock_predictions ............ [[32mSELECT 1[0m in 0.96s]
[2025-05-12T16:22:42.713+0700] {subprocess.py:93} INFO - [0m09:22:42  3 of 4 START sql table model public_analytics.lstm_performance_metrics ......... [RUN]
[2025-05-12T16:22:42.718+0700] {subprocess.py:93} INFO - [0m09:22:42  4 of 4 START sql table model public_analytics.stock_prediction_dashboard ....... [RUN]
[2025-05-12T16:22:43.087+0700] {subprocess.py:93} INFO - [0m09:22:43  4 of 4 ERROR creating sql table model public_analytics.stock_prediction_dashboard  [[31mERROR[0m in 0.34s]
[2025-05-12T16:22:43.092+0700] {subprocess.py:93} INFO - [0m09:22:43  3 of 4 OK created sql table model public_analytics.lstm_performance_metrics .... [[32mSELECT 1[0m in 0.36s]
[2025-05-12T16:22:43.237+0700] {subprocess.py:93} INFO - [0m09:22:43
[2025-05-12T16:22:43.242+0700] {subprocess.py:93} INFO - [0m09:22:43  Finished running 3 table models, 1 view model in 0 hours 0 minutes and 3.12 seconds (3.12s).
[2025-05-12T16:22:43.609+0700] {subprocess.py:93} INFO - [0m09:22:43
[2025-05-12T16:22:43.624+0700] {subprocess.py:93} INFO - [0m09:22:43  [31mCompleted with 1 error, 0 partial successes, and 0 warnings:[0m
[2025-05-12T16:22:43.641+0700] {subprocess.py:93} INFO - [0m09:22:43
[2025-05-12T16:22:43.662+0700] {subprocess.py:93} INFO - [0m09:22:43    Database Error in model stock_prediction_dashboard (models/marts/analytics/stock_prediction_dashboard.sql)
[2025-05-12T16:22:43.663+0700] {subprocess.py:93} INFO -   column m.histogram does not exist
[2025-05-12T16:22:43.666+0700] {subprocess.py:93} INFO -   LINE 28:         m.histogram
[2025-05-12T16:22:43.670+0700] {subprocess.py:93} INFO -                    ^
[2025-05-12T16:22:43.671+0700] {subprocess.py:93} INFO -   compiled code at target/run/idx_stock/models/marts/analytics/stock_prediction_dashboard.sql
[2025-05-12T16:22:43.674+0700] {subprocess.py:93} INFO - [0m09:22:43
[2025-05-12T16:22:43.676+0700] {subprocess.py:93} INFO - [0m09:22:43  Done. PASS=3 WARN=0 ERROR=1 SKIP=0 TOTAL=4
[2025-05-12T16:22:47.045+0700] {subprocess.py:97} INFO - Command exited with return code 1
[2025-05-12T16:22:47.047+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-12T16:22:47.095+0700] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-05-12T16:22:47.110+0700] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=bbca_lstm_prediction, task_id=run_lstm_dbt, run_id=scheduled__2025-05-08T11:30:00+00:00, execution_date=20250508T113000, start_date=20250512T092228, end_date=20250512T092247
[2025-05-12T16:22:47.152+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 131 for task run_lstm_dbt (Bash command failed. The command returned a non-zero exit code 1.; 5072)
[2025-05-12T16:22:47.182+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-05-12T16:22:47.233+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-12T16:22:47.240+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
